.github/workflows/build-test-e2e.yml

```diff
@@ -0,0 +1,52 @@
+name: 'Build/Run End-to-End Tests'
+
+on: push
+jobs:
+  build-and-run-e2e:
+    if: ${{ github.ref == 'develop' || github.ref == 'release-**' || github.actor == github.repository_owner }}
+    runs-on: macos-latest
+    env:
+      CONTAINER_NAME: 'swc-tomcat'
+      CI: true
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup docker (missing on MacOS)
+        if: runner.os == 'macos'
+        run: |
+          # workaround for https://github.com/actions/runner-images/issues/8104
+          export HOMEBREW_NO_INSTALLED_DEPENDENTS_CHECK=1
+          brew update
+          brew reinstall qemu
+
+          brew install docker
+          colima start
+      - name: Build SWC
+        uses: ./.github/workflows/build
+      - name: Run SWC on Tomcat
+        run: |
+          docker run \
+            --name "${CONTAINER_NAME}" \
+            -d --rm \
+            -p 8888:8080 \
+            -v "/$(pwd)/target/portal-develop-SNAPSHOT.war:/usr/local/tomcat/webapps/ROOT.war" \
```

Comment:
Rename to`ROOT.war` so tomcat serves app at the base URL, rather than including the war name in the URL.